popup.js

```diff
@@ -135,6 +145,66 @@ function minimizeStyles(message) {
 
 /**
  * Removes all style attribute properties that are unneeded for a single
+ *     pseudo element.
+ *
+ * @param {Object} message The message Object that contains the pseudo element
+ *     whose style attributes should be minimized.
+ * @param {Document} doc The Document that contains the rendered HTML.
+ * @param {string} selector The CSS selector for the pseudo element.
+ * @param {Array<string>} finalPseudoElements An array to contain the final
+ *     declaration of pseudo elements.  It will be updated to reflect the pseudo
+ *     element that is being processed.
+ */
+function minimizePseudoElementStyle(
+    message,
+    doc,
+    selector,
+    finalPseudoElements) {
+  var maxNumberOfIterations = 5;
+  var match = selector.match(/^#(.*):(:.*)$/);
+  var id = match[1];
+  var type = match[2];
+  var element = doc.getElementById(id);
+  if (element) {
+    var originalStyleMap = message.pseudoElementSelectorToCSSMap[selector];
+    var requiredStyleMap = {};
+    // We compare the computed style before and after removing the pseudo element
```

Comment:
Can you mention how the pseudo element is removed? I.e., that you remove it by removing the id.